packages/text-input/src/TextInput.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { forwardRef, useState } from 'react';
+import React, { useState } from 'react';
```

Comment:
Is removing this related to the typescript changes?

Reply:
Just removing the explicit import in favor of `React.forwardRef`